### ISSUE TITLE
ignore_checks is not applied in all templates when linting multiple from one command

### ISSUE THREAD

**Issue by kgeisink** [awslabs/cfn-python-lint#428]
*cfn-lint version: 0.8.2* (Python 3.6 if it matters)

*Description of issue.*

I am unable to lint multiple template files together when more than one have the `ignore_checks` metadata set.

I have two YAML templates within the same directory. Each of the templates I've configured to ignore a specific warning like:
```yaml
cfn-lint:
    config:
      ignore_checks:
        - W6001 # The value of output (<output_resource>) is imported from another output (<source_output>)
```
(Each template has a different check I want to ignore, I am not sure whether that is relevant.)

When I run the linter on the whole folder like:
```shell
cfn-lint path/to/templates/*.yaml
```
The warnings of one of the templates are not ignored. I've also tried the other ways of linting multiple files such as
```shell
cfn-lint path/to/templates/template1.yaml path/to/templates/template2.yaml
```
which produces the same result.

The only way that I all warnings are properly ignored is by linting the templates separately so
```shell
cfn-lint path/to/templates/template1.yaml 
cfn-lint path/to/templates/template2.yaml
```

**Comment by kddejong** [comment]
Thanks for reporting this.  I thought I had this working correctly.  I'll go back and walk through it using the process you have to see what the issues are.
